packages/ra-ui-materialui/src/input/RadioButtonGroupInput.tsx

```diff
@@ -277,6 +296,7 @@ export type RadioButtonGroupInputProps = Omit<CommonInputProps, 'source'> &
     RadioGroupProps & {
         options?: RadioGroupProps;
         source?: string;
+        disableValue?: string;
```

Comment:
I believe we forgot to add this prop to the `ChoicesProps`. Can you add it there instead?